pkg/install/v1alpha1/installer.go

```diff
@@ -140,6 +149,44 @@ func (i *simpleInstaller) Install() []error {
 	return i.errors
 }
 
+// cleanResources are the versioned openebs respurces that needs to cleaned
+// after installing the latest versions of the respective resource
+var cleanResources = []resourceMetadata{
+	resourceMetadata{
+		name: "castemplates",
+	},
+	resourceMetadata{
+		name:      "runtasks",
+		namespace: menv.Get(menv.OpenEBSNamespace),
+	},
+}
+
+// Clean is the cleanup function that removes old version of openebs resources
+func (i *simpleInstaller) Clean() error {
+	for _, res := range cleanResources {
```

Comment:
It will be good to avoid cleanup during every api server restarts.

Reply:
Added a label selector to restrict the list of resources that needs cleaning.